bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/item/ItemResource.java

```diff
@@ -385,6 +390,10 @@ public Response removeMember(@PathParam("itemName") @ApiParam(value = "item name
             @ApiResponse(code = 404, message = "Item not found or item is not editable.") })
     public Response removeItem(@PathParam("itemname") @ApiParam(value = "item name", required = true) String itemname) {
 
+        if (itemname == null) {
+            return Response.status(Status.BAD_REQUEST).build();
```

Comment:
Can this situation at all happen? After all, it is a part of the URI path, so how could it be null?
If it can happen, then please also add 400 as a potential response to the swagger annotations of this method.

Reply:
Ah, I didn't realized that it has been marked as a `@PathParam`. Don't know if it is then possible to be null.

Reply:
Looking at this problem again I think you are right. Since `itemname` is part of the path it cannot be `null` here. So I added `@NonNull` to satisfy the IDE :) (which was my original intention by touching this line at all)